react-native.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   project: {
     android: {sourceDir: './example/android'},
     ios: {project: './example/ios/example.xcworkspace'},
+    windows: {project: './example/windows/ClipboardExample.sln'},
```

Comment:
I don't think this will actually work. What happens when you run `react-native-config`?

Since 0.62 doesn't consume the config, and 0.63 does, if you['re going to set this I'd set it according to: https://microsoft.github.io/react-native-windows/docs/config#example-react-nativeconfigjs-for-a-myapp

Something like:

```json
windows: {
      sourceDir: './example/windows',
      solutionFile: 'ClipboardExample.sln',
      project: {
        projectFile: 'ClipboardExample/ClipboardExample.vcxproj',
      },
    },
```

Reply:
good catch @jonthysell, right 0.62 isn't using it but I figured I'd put it there for the next person when they migrate to 63 if it ever needs to :)